rosidl_typesupport_opendds_c/CMakeLists.txt

```diff
@@ -14,14 +14,10 @@ endif()
 
 # Load required CMake packages.
 find_package(ament_cmake REQUIRED)
-
 find_package(opendds_cmake_module REQUIRED)
+find_package(OpenDDS QUIET)
 
-find_package(OpenDDS QUIET MODULE)
-if(OpenDDS_FOUND)
-  find_package(OpenDDS MODULE)
-endif()
-if(NOT OpenDDS_FOUND OR OPENDDS_STATIC_DISABLE)
+if(NOT OpenDDS_FOUND)
   ament_package()
   if(NOT OpenDDS_FOUND)
```

Comment:
Since you got rid of `OPENDDS_STATIC_DISABLE`, also get rid of this if else because it was handling the difference between `NOT OpenDDS_FOUND` and `OPENDDS_STATIC_DISABLE` for what message to give. Something like:

```cmake
  if(NOT OpenDDS_FOUND)
    ament_package()
    message(WARNING "Could not find OpenDDS - skipping '${PROJECT_NAME}'")
    return()
  endif()
```

I would suggest it properly but GitHub doesn't allow suggestions more than a few lines away from changes.

Reply:
Done